appserver/packager/appserver-base/src/main/docs/README.txt

```diff
@@ -8,9 +8,9 @@ Here are a few short steps to get you started...
 
 Payara Server currently supports the following Java Virtual Machines:
 
-* Oracle JDK8 (u72+)
-* Azul Zulu JDK8 (u72+)
-* OpenJDK JDK8 (u72+)
+* Oracle JDK8 (u231+), Oracle JDK 11 (11.0.5+)
+* Azul Zulu JDK8 (u232+), Azul Zulu JDK 11 (11.0.5u10+)
+* OpenJDK JDK8 (u232+), OpenJDK 11 (11.0.5+)
 
 For IBM J9 support, please download Payara Blue from http://www.payara.fish/downloads
```

Comment:
Slightly ancillary, but this should probably be removed tbh - there is no payara blue for 5.x.

Reply:
Thanks, that's what I wanted to hear! :)